packages/m2m-gateway/src/model/errors.ts

```diff
@@ -48,6 +48,7 @@ export const errorCodes = {
   eserviceRiskAnalysisNotFound: "0030",
   eserviceTemplateRiskAnalysisNotFound: "0031",
   delegationEServiceMismatch: "0032",
+  eserviceTemplateVersionNotInSuspendedState: "0033",
```

Comment:
Unneeded, see: https://github.com/pagopa/interop-be-monorepo/pull/2407#discussion_r2378242011
```suggestion
```